src/app/admin/all_employees/[id]/layout.tsx

```diff
@@ -0,0 +1,66 @@
+import { Avatar, Button } from "@/components";
+import {
+  BriefcaseIcon,
+  EnvelopeIcon,
+  PencilIcon,
+  UserCircleIcon,
+} from "@heroicons/react/24/outline";
+import SingleEmployeeSideBar from "./SingleEmployeeSideBar";
+
+const SingleEmployeeHeader = () => {
+  return (
+    <section className="relative flex flex-col xs:flex-row">
+      <div className="mditems-center flex gap-1  px-2">
+        <div className="w-1/4">
+          <Avatar
+            imageUrl="/images/girl.jpg"
+            alt=""
+            rounded="md"
+            className="vs h-[4rem] w-[3rem] flex-1"
+          />
+        </div>
+        <div className="flex flex-col gap-2">
+          <h2 className="text-lg font-bold">Brooklyn Simmons</h2>
+          <p className="flex items-center gap-2 text-xs xs:text-sm">
+            <span>
+              <BriefcaseIcon className="h-6 w-6" />
+            </span>
+            Project Manager
+          </p>
+          <p className="flex items-center gap-2 text-xs xs:text-sm">
+            <span>
+              <EnvelopeIcon className="h-6 w-6" />
+            </span>
+            brooklyn.s@example.com
+          </p>
+        </div>
+      </div>
+      <Button
```

Comment:
I think we should reduce the size of this button

Reply:
Alright I will do that when I want to push after completing the remaining parta